config/routes.rb

```diff
@@ -4,5 +4,5 @@
 
   # Defines the root path route ("/")
   # root "posts#index"
-  resources :payments, only: [ :index, :create, :show], param: :id
+  resources :payments, only: [ :index, :create, :show]
```

Comment:
Prefer to format like:
`resources :payments, only: [:index, :create, :show]`

Reply:
amended, thanks Joe